sources/packages/api/src-sql/MSFAANumbers/Add-col-offering-cancelled.sql

```diff
@@ -0,0 +1,27 @@
+-- Create offering_intensity
```

Comment:
Just a suggestion, the file name and the columns added are slightly different. Can give a more general file name like Add-cols-for-msfaa or Add-cols.

Reply:
Yeah even I was skeptical, because I was doing it as an enhancement for MSFAA and also cancelled file changes in the response file. I will change it